pkg/utils/build/quay.go

```diff
@@ -103,3 +104,35 @@ func DoesTagExistsInQuay(imageURL string) (bool, error) {
 	}
 	return false, nil
 }
+
+func IsImageRepoPublic(quayImageRepoName string) (bool, error) {
+	isPublic, err := quayClient.IsRepositoryPublic(quayOrg, quayImageRepoName)
+	if isPublic {
+		return true, nil
+	} else if !isPublic && err == nil {
+		return false, nil
+	} else {
+		return false, err
+	}
+}
+
+func IsQuayOrgSupportsPrivateRepo() (bool, error) {
+	repositoryRequest := quay.RepositoryRequest{
+		Namespace:   quayOrg,
+		Visibility:  "private",
+		Description: "Test private repository",
+		Repository:  constants.SamplePrivateRepoName,
+	}
+	repo, err := quayClient.CreateRepository(repositoryRequest)
```

Comment:
What would clean this repository up?

Reply:
Cleaning up of the sample private repo created is done in the AfterAll block [here](https://github.com/redhat-appstudio/e2e-tests/pull/716/files#diff-938ddf0d569d68c3df496d8b4065614ff56cc303c33555755621cffcec97ef74R135)